edx_proctoring/templates/proctored_exam/visit_exam_content.html

```diff
@@ -35,10 +35,4 @@
       }
     );
   </script>
-{% elif has_due_date %}
```

Comment:
I created https://openedx.atlassian.net/browse/EDUCATOR-405 to do a cleanup around when exam content is available after due date.